backend/src/core/middlewares/file-attachment.middleware.ts

```diff
@@ -48,7 +51,36 @@ function preprocessPotentialIncomingFile(
   next()
 }
 
+// allow attachment only if (1) the user is not using default email address; (2) user's email has been added to the email_from_address table
+// attachment is a high-cost and high-risk feature that we want to limit to users who have been vetted
+async function checkAttachmentPermission(
+  req: Request,
+  res: Response,
+  next: NextFunction
+): Promise<void> {
+  if (req.files?.attachments) {
+    const { from } = req.body
+    if (isDefaultFromAddress(from)) {
+      res.status(403).json({
+        message: 'Attachments are not allowed for default email address',
```

Comment:
Maybe we can add a bit more info here `Attachments are not allowed for Postman default from-email-address`.

Reply:
done